pages/mesosphere/dcos/services/kubernetes/2.5.0-1.16.9/changelog/index.md

```diff
@@ -106,7 +106,7 @@ excerpt: Changelog for DC/OS Kubernetes
 
 * Kubernetes 1.15.3
 * dcos-commons 0.56.3
-* Adds [configuration options to enable Kubernetes auditing](/mesosphere/dcos/services/kubernetes/2.5.0-1.16.9/kubernetes-audit/). Kubernetes auditing provides a security-relevant chronological set of records. These document the sequence of activities that have affected the system by users, administrators or other components of the system.
+* Adds [configuration options to enable Kubernetes auditing](/mesosphere/dcos/services/kubernetes/2.5.0-1.16.9/operations/kubernetes-audit/). Kubernetes auditing provides a security-relevant chronological set of records. These document the sequence of activities that have affected the system by users, administrators or other components of the system.
```

Comment:
WOW, good catch! I thought we had the 404 detection script 🤔

Reply:
> WOW, good catch! I thought we had the 404 detection script 🤔

the link catcher has been broken since before I got here.